geoapi/routes/projects.py

```diff
@@ -208,9 +208,15 @@ def get(self, projectId: int):
     @project_admin_or_creator_permissions
     def delete(self, projectId: int):
         u = request.current_user
-        logger.info("Delete project:{} for user:{}".format(projectId,
-                                                           u.username))
-        return ProjectsService.delete(db_session, u, projectId)
+        # Retrieve the project using the projectId to get its UUID
+        project = ProjectsService.get(db_session, project_id=projectId, user=u)
+        # Check if the project exists and log the information including the UUID
+        if project:
+            logger.info("Delete project:{} with UUID:{} for user:{}".format(
+                projectId, project.uuid, u.username))
+            return ProjectsService.delete(db_session, u, projectId)
+        else:
+            abort(404, "Project not found")
```

Comment:
good thought on adding a check related to the project not being found.  but it can be dropped in this instance as the `project_admin_or_creator_permissions` decorator has that logic in a `check_access_and_get_project` method that it calls.